diluvian/coord.py

```diff
@@ -0,0 +1,400 @@
+from keras.engine import Layer, InputSpec
```

Comment:
There used to be a `diluvian/third_party` folder, which would be a better place to this.

Reply:
I went ahead and moved it